source/fundamentals/connection/connect.txt

```diff
@@ -57,10 +57,23 @@ options, skip to the :ref:`node-connection-options` section.
 
 .. _connect-sample-node-driver:
 
-The code below shows how you can use a connection URI in a client to
-connect to MongoDB.
+Atlas Connection Example
+------------------------
 
-.. literalinclude:: /code-snippets/connection/connect.js
+To connect to a MongoDB deployment on Atlas you must first create a client. You
+can create a client that uses your connection string and any other
+client options by passing your URI and a ``MongoClientOptions`` object to a
+``MongoClient``.
+
+You can set the Stable API version client option to avoid breaking changes when
```

Comment:
Confusing that the option is called `serverApi` in code :/

I wonder if this could be phrased more directly; instead of telling people why we're using Stable API (avoiding breaking changes), we can just tell them what this option is doing:

"The ``serverApi`` option in ``MongoClientOptions`` enables the Stable API feature, which forces the server to run operations with behavior compatible with the specified API version. [skip the Stable API link] The following code shows how..."

[Stable API link in admonition at bottom]

Again, happy to workshop!

Reply:
Changed the wording slightly from your suggestion, but wrote something pretty similar. Let me know if you feel that's a good middle ground or not